tests/github-glue.test.ts

```diff
@@ -10,26 +10,26 @@ branch to be deleted on github and the local repo.  The test will
 attempt cleanup of a previously failed test.
 
 Setup:
-gitgitgadget.githubtest.githubuser must be configured for this test to
+gitgitgadget.githubTest.gitHubUser must be configured for this test to
```

Comment:
I won't deny there may be inconsistencies in the current code but shouldn't it be moving to use lower case syntax?  i.e. config vars are case-insensitive.

Reply:
Config variable names are indeed case-sensitive, which is why we can camelCase them. And that makes cSpell _a lot_ happier. (Which makes me happier, because then I can see the actual typos really easily, such as "commmand").

Reply:
> Config variable names are indeed case-sensitive

I accept the reason for camelCase but just wanted to clarify config variable names are case-**insensitive**.

Reply:
Issue #348 opened to resolve a problem here.  Two part config variable names are case-insensitive but for three-part (has a subsection), part two is case sensitive.  In line 13 above, `githubTest` is case sensitive.